ConverterService/pkg/repository/currencyManipulation.go

```diff
@@ -0,0 +1,57 @@
+package repository
+
+import (
+	"context"
+	"fmt"
+	"github.com/jmoiron/sqlx"
+	"time"
+)
+
+type currenciesInfo interface {
+	GetAllPairs(ctx context.Context) ([]Currencies, error)
+	ConvertMoney(ctx context.Context, baseCurrency string, quoteCurrency string, value float64) (float64, error)
+	AddNewPair(ctx context.Context, baseCurrency string, quoteCurrency string, exchangeRate float64) (int, error)
+}
+
+type CurrencyDb struct {
+	db *sqlx.DB
+}
+
+func NewCurrencyDb(db *sqlx.DB) *CurrencyDb {
+	return &CurrencyDb{db: db}
+}
+
+func (c *CurrencyDb) GetAllPairs(ctx context.Context) (pairs []Currencies, err error) {
+	//передача контекста
+	query := "SELECT cur.id, cur.CurrencyFrom, cur.CurrencyTo from currency_exchange cur"
+	return pairs, c.db.Select(
+		&pairs,
+		query,
+	)
+}
+
+func (c *CurrencyDb) ConvertMoney(ctx context.Context, baseCurrency string, quoteCurrency string, amount float64) (exchangeRate float64, err error) {
+	query := fmt.Sprintf(`SELECT cur.ExchangeRate from currency_exchange cur
+											where currencyFrom=$1 and currencyTo=$2`)
```

Comment:
Можно делать так, для того, чтобы всегда влезал весь SQL запрос:
```suggestion
	query := fmt.Sprintf(`
SELECT 
    cur.ExchangeRate 
FROM currency_exchange cur
WHERE currencyFrom=$1 and currencyTo=$2
       `)
```

ну и писать SQL нужно либо в верхнем либо в нижнем регистре